modules/config.xqm

```diff
@@ -24,7 +24,7 @@ declare namespace tei = "http://www.tei-c.org/ns/1.0";
  :)
 declare variable $config:origin-whitelist := (
     "(?:https?://localhost:.*|https?://127.0.0.1:.*)",
-    "https?://unpkg.com",
+    "https?://jsdelivr.net",
```

Comment:
why remove unpkg altogether here? I think it's better to keep it in case someone needs to switch when jsdelivr has trouble